src/renderer/api-decorator.js

```diff
@@ -319,6 +319,11 @@
 
     // check if a license key is valid or not
     if (!isLicenseKeyValid()) {
+        // log invalid license key info in debug.log
+        asyncApiCall('write-to-log', {
+            level: 'info',
+            message: `[licenseKey] [${initialOptions.uuid} - ${initialOptions.name}]: invalid OpenFin license key`
```

Comment:
should we include `${licenseKey }` in there just in case there is a malformed one we would be able to see it